ArduPlane/ArduPlane.cpp

```diff
@@ -310,6 +310,12 @@ void Plane::one_second_loop()
     // indicates that the sensor or subsystem is present but not
     // functioning correctly
     gcs().update_sensor_status_flags();
+
+#if HAL_WITH_IO_MCU
+    if (should_log(MASK_LOG_PM)) {
```

Comment:
This should probably be `MASK_LOG_RC` on some vehicles and `MASK_LOG_RCIN|MASK_LOG_RCOU` on others, I think?

Reply:
I could argue either way. I selected PM, because this is basically failure monitoring of the communication health. I can be convinced otherwise :)